src/forms/Checkbox.jsx

```diff
@@ -14,12 +20,22 @@ class Checkbox extends React.Component {
 			checked: props.checked || false
 		};
 		this.onChange = this.onChange.bind(this);
+		this.onBlur = this.onBlur.bind(this);
+		this.onFocus = this.onFocus.bind(this);
 	}
 
 	onChange(e) {
 		this.setState({ checked: e.target.checked });
 	}
```

Comment:
we will want to move away from this being stateful into something stateless, but that can be done in a different PR.